src/styles/blocks/Search-box-small.css

```diff
@@ -0,0 +1,44 @@
+.small_search-bar {
```

Comment:
File's name should match the class name of the block.
Also, there is no need to create two different blocks. It's just the same block with the default state (small) and the big state